src/content/guides/code-splitting.md

```diff
@@ -263,7 +264,7 @@ lodash.bundle.js   541 kB       0  [emitted]  [big]  lodash
    [3] (webpack)/buildin/module.js 517 bytes {0} [built]
 ```
 
-If you've enabled [`async` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) via a pre-processor like babel, note that you can simplify the code as `import()` statements just return promises:
+If you've enabled [`async` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) via a pre-processor like Babel + [Syntax Dynamic Import Babel Plugin](https://babeljs.io/docs/plugins/syntax-dynamic-import/#installation), note that you can simplify the code as `import()` statements just return promises:
```

Comment:
@kcolton Just to make this a little more 'human', could I suggest…

> Babel and the [Syntax Dynamic Import Babel Plugin](https://babeljs.io/docs/plugins/syntax-dynamic-import/#installation)

Also, I know it's not your words originally, but while we're here :)

> note that you can simplify the code as `import()` statements return promises:

Let's remove that unnecessary 'just'.

Reply:
@ChrisChinchilla Sounds good to me! 👍  

I actually removed "just" and then figured I would try to minimize changes haha. I'll modify with your suggestions. Coming right up.

Before deciding to minimize the diff, thought about rephrasing entirely. The statement read backwards to me; talking about the means before the ends. Here was what I originally wrote (incorporating the more 'human' suggestion). 

I didn't think it drastically changed comprehension or usefulness so didn't bother with it.

---

As `import()` returns a promise, it can be used with [`async` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). However, this requires using a pre-processor like Babel and the [Syntax Dynamic Import Babel Plugin](https://babeljs.io/docs/plugins/syntax-dynamic-import/#installation).